omero/developers/index.txt

```diff
@@ -115,6 +115,7 @@ Web
     Web/WritingViews
     Web/WritingTemplates
     Web/CSRF
+    Web/Webclient
```

Comment:
Sure, one it grows probably we need to have new section, because there will be need for subsections.